CONTRIBUTING.md

```diff
@@ -12,7 +12,68 @@ Please consult the [README](./README.md) file at the root of this repository.
 
 ## How to Contribute
 
-Please follow the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website.
+Please read the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website. It contains some general instructions that should be followed when contributing to any of the Creative Commons open-source repositories.
+
+### Bugs
+
+If you find a bug, please open an issue in this repository describing the bug. The bug description should contain:
+
+* Step-by-step to reproduce
+* What the expected behaviour is
+* Which browsers, including their versions, and operating system were used to reproduce the bug
+
+Example of a good bug description:
```

Comment:
This is covered in our bug template that we already have set up when filing an issue on GitHub and these instructions are a little bit different from the template.